conda/seacas/meta.yaml

```diff
@@ -0,0 +1,59 @@
+# MAKE SURE THAT seacas_version and seacas_git_rev match if performing a version update!
+{% set build = "0" %}
+{% set seacas_version = "2022.05.16" %}
+{% set seacas_git_rev = "v2022-05-16" %}
+{% set strbuild = "build_" + build|string %}
+
+package:
+  name: moose-seacas
+  version: {{ seacas_version }}
+
+source:
+  git_url: https://github.com/sandialabs/seacas
+  git_rev: {{ seacas_git_rev }}
+  patches:
+    - use-curl-instead-of-wget.patch
+
+build:
+  number: {{ build }}
+  string: {{ strbuild }}
+  script_env:
+    - MOOSE_JOBS
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('fortran') }}
+    - cmake
+    - make
+    - automake
+  run:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('fortran') }}
+    - python 3
+
+test:
+  commands:
+    - exodiff --version
+    - algebra
+    - explore
+    - grepos
+    - gjoin -v
```

Comment:
As I recall, the Bison team uses: `blot`... or was it `xblot`? @bwspenc

Reply:
Dang:

HAVE_X11 | YES, NO | YES | Does the system have X11 libraries and include files; used for blot, fastq
-- | -- | -- | --

Reply:
Since this will be a package which will never depend on, or be depending on any of our moose-* packages... I think we can support X11 (we do it for moose-peacock). I'll investigate.

Reply:
My investigation led to this - `xorg-libx11` was not a sufficient package for enabling this, for either Mac or Linux. Hopefully you can find a better way.

Reply:
On Mac, it failed while building components of Zoltan, but gave very little output. Perhaps there's a verbose mode I can turn on?

```
[ 38%] Linking C shared library libsimpi.dylib
[ 38%] Built target exodus
[ 38%] Built target simpi
make: *** [Makefile:166: all] Error 2
Traceback (most recent call last):
  File "/Users/user/mambaforge3/envs/build/bin/conda-mambabuild", line 10, in <module>
    sys.exit(main())
  File "/Users/user/mambaforge3/envs/build/lib/python3.10/site-packages/boa/cli/mambabuild.py", line 239, in main
    call_conda_build(action, config)
  File "/Users/user/mambaforge3/envs/build/lib/python3.10/site-packages/boa/cli/mambabuild.py", line 211, in call_conda_build
    result = api.build(
  File "/Users/user/mambaforge3/envs/build/lib/python3.10/site-packages/conda_build/api.py", line 186, in build
    return build_tree(
  File "/Users/user/mambaforge3/envs/build/lib/python3.10/site-packages/conda_build/build.py", line 3088, in build_tree
    packages_from_this = build(metadata, stats,
  File "/Users/user/mambaforge3/envs/build/lib/python3.10/site-packages/conda_build/build.py", line 2211, in build
    utils.check_call_env(cmd, env=env, rewrite_stdout_env=rewrite_env,
  File "/Users/user/mambaforge3/envs/build/lib/python3.10/site-packages/conda_build/utils.py", line 411, in check_call_env
    return _func_defaulting_env_to_os_environ('call', *popenargs, **kwargs)
  File "/Users/user/mambaforge3/envs/build/lib/python3.10/site-packages/conda_build/utils.py", line 391, in _func_defaulting_env_to_os_environ
    raise subprocess.CalledProcessError(proc.returncode, _args)
subprocess.CalledProcessError: Command '['/bin/bash', '-o', 'errexit', '/Users/user/mambaforge3/envs/build/conda-bld/moose-seacas_1658772413078/work/conda_build.sh']' returned non-zero exit status 2.
````

Reply:
yeah I get that error as well. If you scroll up a ways (a way ways lol), you'll see the real error:

```pre
/Users/milljm/mambaforge3/conda-bld/moose-seacas_1658773530447/work/packages/seacas/libraries/svdi/cgi/x11_vdix11.c:428:10: fatal error: 'X11/Xatom.h' file not found
#include <X11/Xatom.h>
         ^~~~~~~~~~~~~
1 error generated.
```
of the like. Yet, performing a `find ` for said missing file... there it sits, right where I'd expect it:

```pre
$CONDA_PREFIX/include/X11/<right here>
```

So far I am chalking this up to a faulty cmake config. But I am still investigating. Building is slow-going on my laptop.

Reply:
```pre
Processing enabled TPL: X11 (enabled explicitly, disable with -DTPL_ENABLE_X11=OFF)
-- X11_LIBRARY_NAMES='X11'
-- Searching for libs in X11_LIBRARY_DIRS=''
-- Searching for a lib in the set "X11":
--   Searching for lib 'X11' ...
--     Found lib '/Users/milljm/mambaforge3/conda-bld/moose-seacas_1658774488020/_build_env/lib/libX11.dylib'
-- TPL_X11_LIBRARIES='/Users/milljm/mambaforge3/conda-bld/moose-seacas_1658774488020/_build_env/lib/libX11.dylib'
-- TPL_X11_INCLUDE_DIRS='/opt/X11/include'
```
maybe?

Reply:
Well that's interesting......you would think that once it found the lib, it would use that location to find the includes....

Reply:
We should be able to manually define those using `-DTPL_X11_LIBRARIES=` and `-DTPL_X11_INCLUDE_DIRS=` during the `cmake-config` step.

Reply:
Alright, I've got it working, I think. Will push up a new commit soon.

Reply:
Addressed in 8d844a9